net/tailscale/Makefile

```diff
@@ -42,41 +42,29 @@ define Package/tailscale/Default
   DEPENDS:=$(GO_ARCH_DEPENDS)
 endef
 
-define Package/tailscaled
-  $(call Package/tailscale/Default)
-  TITLE+= (daemon)
-  DEPENDS+= +ca-bundle +kmod-tun
-endef
-
 define Package/tailscale
   $(call Package/tailscale/Default)
```

Comment:
so you can remove `Package/tailscale/Default` and change it to:
```
define Package/tailscale
  SECTION:=net
  CATEGORY:=Network
  SUBMENU:=VPN
  TITLE:=Zero config VPN
  URL:=https://tailscale.com
  DEPENDS:=$(GO_ARCH_DEPENDS) +ca-bundle +kmod-tun
  PROVIDES:=tailscale tailscaled
endef
```

Reply:
Thank you @1715173329 for the helpful tip! I realized that the 'Default' section is not needed anymore since only one package is created. I will make the changes you recommended and force push.